internetarchive/openlibrary
# Show toolbar and follow button for list visitor

Closes #9337 


Fix

### Technical

Remove ```if/else``` condition, uses the ```lst['key']``` to determine if the user is the owner of the list and show relevant components within the ```account/view.html``` template.

### Testing


- [Create two user accounts](https://github.com/internetarchive/openlibrary/wiki/Creating-and-Logging-in-as-a-new-user-on-your-local-client) and use one account to create a list of books
- Navigate to the list page and check that the follow button does not appear within the list of lists and single list webpages.
- Sign out of the list creator account and sign into the other newly created account.
- Navigate to the created book list, verify that the follow button appears on the list page and that pressing follow will add the creator's account to the visitor's ```Following``` tab.

### Screenshot

![image](https://github.com/user-attachments/assets/d078aa37-9fb5-4790-8ad6-0cc49c532522)
![image](https://github.com/user-attachments/assets/af22bc64-67bc-4e43-99f2-6b73ad2c6e91)
![image](https://github.com/user-attachments/assets/7d3bb076-4a45-4ef4-b6e9-f5fc496e170d)
![image](https://github.com/user-attachments/assets/4c39180d-e4e4-4478-9d94-f9185d6442ea)



### Stakeholders

@jimchamp
